dcos/nodeutil/util_test.go

```diff
@@ -301,3 +301,14 @@ func TestContextWithHeaders(t *testing.T) {
 		t.Fatalf("Expect header `TEST:123`. Got %+v", headerFromContext)
 	}
 }
+
+func TestFindCompletedFramework(t *testing.T) {
```

Comment:
I think we should add a test which searches for a completed task that lives somewhere in `state.Frameworks` and another test that searches for a completed task somewhere in `state.CompletedFrameworks` to prevent a regression later on.